micro_dl/light/data.py

```diff
@@ -1,73 +1,81 @@
-from typing import Callable, Literal, Union
+import logging
+import os
+import tempfile
+from typing import Any, Callable, Literal, Union
 
 import numpy as np
 import torch
-from iohub.ngff import ImageArray, Plate, open_ome_zarr
+import zarr
+from iohub.ngff import ImageArray, Plate, Position, open_ome_zarr
 from lightning.pytorch import LightningDataModule
+from monai.data import set_track_meta
 from monai.transforms import (
     CenterSpatialCropd,
     Compose,
-    NormalizeIntensity,
+    MapTransform,
     RandAdjustContrastd,
     RandAffined,
     RandGaussianSmoothd,
     RandWeightedCropd,
 )
-from torch.utils.data import DataLoader, Dataset, Subset
+from numpy.typing import NDArray
+from torch.utils.data import DataLoader, Dataset
+
+
+class NormalizeTargetd(MapTransform):
```

Comment:
The code is working well. Please add more verobse signatures, docstrings, and comments to entire module.

Reply:
See 55aabae6e3cdd91f2aa402af85031636d522ad78